content/post/2019-06-11-cran-checks-api.Rmarkdown

```diff
@@ -0,0 +1,72 @@
+---
+title:  Overview of the CRAN checks API
+date: '2019-06-05'
+slug:  cran-checks-api
+tags:
+  - CRAN
+---
+
+We've recently introduced [CRAN checks on our blog](/2019/04/25/r-devel-linux-x86-64-debian-clang/): once on CRAN your package is checked daily on a dozen platforms, and new failures can lead to its being archived or orphaned. We mentioned [ways to monitor your CRAN checks results](/2019/04/25/r-devel-linux-x86-64-debian-clang/#cran-checks-surveillance), including a cool web API, the CRAN checks API. We were thrilled to talk to its creator Scott Chamberlain, co-founder and tech lead at rOpenSci!  :telephone_receiver: :tada: In this post, we shall summarize our discussion with him about the API's functionalities, tech stack and future goals!
+
+# CRAN checks data at your fingertips
+
+### A handy API for all CRAN package maintainers
+
+When [introducing the API on his blog in 2017](https://recology.info/2017/09/cranchecks-api/), Scott presented the motivation for it with these words _"it’d be nice to have a modern way (read: an API) to check CRAN check results."_. Prior to the API creation, the data was only available via CRAN check pages like e.g. [`rhub` CRAN check results](https://cran.r-project.org/web/checks/check_results_rhub.html), [by package or by maintainer](https://cran.r-project.org/web/checks/). Programmatic access to this data was only possible via webscraping, so now Scott webscrapes it for everyone and makes it available in :sparkles: _JSON_ :sparkles: and :sparkles: _SVG badges_ :sparkles:.
```

Comment:
Instead of "modern" I would say "programmatic" if that's OK.

Reply:
But it's a quote from Scott's blog so we can't edit it :wink:

Reply:
Well, all right then.